src/constants.ts

```diff
@@ -1,14 +1,18 @@
+import path from 'path';
 import { resolveApp } from './utils';
 
+const cache = resolveApp('node_modules/.cache');
+
 export const paths = {
   appPackageJson: resolveApp('package.json'),
   testsSetup: resolveApp('test/setupTests.ts'),
   appRoot: resolveApp('.'),
-  cache: resolveApp('node_modules/.cache'),
+  cache,
```

Comment:
where is this used?

Reply:
@jaredpalmer no anymore since I moved the storage path into this file.

Do you want me to remove it?

I just left it there so people use it when they need it